front-end/src/components/analysis-page/AnalysisPage.test.js

```diff
@@ -21,8 +21,10 @@ describe('AnalysisPage', () => {
         </Provider>
     );
 
-    it('should dispatch openMainPage when Return to Main Page button is clicked', () => {
+
+    it('should dispatch openMainPage when logo button is clicked', () => {
         renderComponent(props);
+
```

Comment:
Nit we shouldnt add these lines

Reply:
yeah sorry I was doing the merge conflict on web I didn't realize :))